rfc/php_technical_committee.wiki

```diff
@@ -52,83 +51,121 @@ For example if there is a proposal to add accessors for PHP classes, then everyt
+a call for nominations.  The nomination period lasts for one week.
 
-==== TC membership ====
+During that week, any eligible core developer may self-nominate by posting to the Internals mailing list.  If the
+candidate is paid to work on php-src or PHP extensions, they must state in their nomination which company or organization
+they work for.  Failure to do so will result in disqualification, as determined by the TC Secretary.
+
+Once nominations are over, the voting process will begin.  Eligible voters are the same as for any RFC.
+
+If there are five or fewer nominees, there is no vote and all nominated individuals are elected.
+
+If there are six or more nominees, a two week vote using the Single Transferable Vote method will take place.  The five
+winning candidates at the end of the vote are elected.
+
+The newly elected TC members will elect from their own number a TC Secretary by simple majority.
+
+=== Vacancies ===
+
+Any TC member may resign at any time via an email to the Internals mailing list.  If the resigning member is the TC
+Secretary, then the TC will elect a new Secretary from its number.
 
-The TC candidates nominate themself to become a TC member. They must be a core developers and should have a general
-knowledge about PHP internals. If the candidates are paid to work on php-src or PHP extensions, they must state in
-their nomination which company or organization they work for.
+If the number of TC members is reduced to three or fewer and there are more than two months until the next regularly
+scheduled election, the Secretary must call for a special election for the vacant seats.  The election process is the
+same as for any other election, but the newly elected TC members will serve only until the next regular election.
 
-The TC shoudl be composed of 5 members. If there are not enough candiates, it can be composed of reduced number of
-members but it must have at least 3 members. The TC election of memebers will happen every year together with
-release managers selection. The RFC voting process using a single transferable vote will be used if more than 5
-candiates volunteer. Otherwise the candidates will be selected automatically.
+==== Work flow ====
+
+The TC is primarily a reactive body, and is not expected to proactively search PRs for issues.  However, TC members
+are permitted to comment on any open issue, and contributors are encouraged to give their comments appropriate weight.
+
+Should a dispute or question about a change arise, the TC may be called on to resolve it.
 
-One member will be selected by other TC members as a TC secratary. The TC secratary is responsible for preparing a vote,
-and communicating the decision. The secretary also have a casting vote if the TC decision vote is a draw.
+  - Any developer (including a TC member) may ask for TC resolution of a issue by adding the label `tc-hold` to it and
```

Comment:
I would prefer to limit it to core developers to prevent some random people asking about resolution...
```suggestion
  - Any core developer (including a TC member) may ask for TC resolution of an issue by adding the label `tc-hold` to it and
```